third-party/eazy/v1/openapi.yaml

```
@@ -1437,10 +1437,12 @@ paths:
             example: 'contactJid@company.com'
         - in: body
           name: message
-          description: Color of the label
+          description: The message to be sent
```

```suggestion
          description: Outgoing message
```
Just suggestion, ignore if you want, both are good :)